pkg/config/validation/validation.go

```diff
@@ -81,6 +81,9 @@ const (
 
 	matchExact  = "exact:"
 	matchPrefix = "prefix:"
+
+	defaultGatewayGroup = "gateway.networking.k8s.io"
```

Comment:
Does it make sense to go ahead and convert these to GVKs and have a set of valid ones? Eventually, we will probably allow for binding to more than just a Gateway

Reply:
Are you suggesting that in place of these default constants for group and kind, we add a constant for a default gvk? Something like this:

```go
gatewayGVK = config.GroupVersionKind{
    Group:   "gateway.networking.k8s.io",
    Kind:    "Gateway",
}
```

I'm not sure a set/map is needed right now until we decide to have more valid GVKs. Seems like a slightly premature optimization. I'd prefer not to add a data structure with one item in it for now and punt that task to later on when we need to have multiple valid GVKs stored somewhere. I'd be ok with making this GVK object though and using that instead of these 2 strings I currently have. Wdyt?

Reply:
No strong preference on my end; whichever you think is best!